src/modality-specific-files/motion.md

```diff
@@ -0,0 +1,247 @@
+# Motion
+
+<!--Here insert link to actual manuscript?-->
+
+For information on how to cite this extension when referencing it in the context of the academic literature, please read [Citing BIDS](../introduction.md#citing-bids).
+on how to appropriately credit this extension when referring to it in the
+context of the academic literature.
+
+This specification has been used to format a number of [example Motion datasets](https://github.com/bids-standard/bids-examples), which can be used as helpful guidance when curating new datasets.
+
+## Motion recording data
+
+{{ MACROS___make_filename_template(
+"raw",
+datatypes=["motion"],
+suffixes=["motion", "channels", "events"])
+}}
+
+Template:
+
+```markdown
+└─ sub-<label>\
+  └─ \[ses-<label>]\
+    └─ motion\
+   ├─ sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>]_tracksys-<label>_motion.tsv
+      ├─ sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>]_tracksys-<label>_motion.json
+      ├─ sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>]_tracksys-<label>_channels.tsv
+      ├─ sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>]_tracksys-<label>_events.tsv
+      └─ sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>]_tracksys-<label>_events.json
+```
+
```

Comment:
Add some very brief and basic description of what are motion capture systems for non experts, important parts and use (e.g. what's a tracking point, etc)